code/modules/antagonists/special_agent/equipment/advanced_emag.dm

```diff
@@ -0,0 +1,17 @@
+/*! 
+This item is an EMAG but it looks and behaves just like an ID card.
+*/
+
+/obj/item/card/id/advemag
```

Comment:
could this just be some vars on card/id?

Reply:
possibly but i think it would end up creating more code since I would need to modify a few more methods on the ID file as well as adding the EMAG interact. This also allows it to be repurposed as an uplink item later if need be.

Reply:
Actually it might be prudent to add this as a HoP exclusive high TC traitor item.